.mci.yml

```diff
@@ -1274,23 +1274,6 @@ buildvariants:
       tasks:
           - name: test_mongohouse
 
-    - name: ubuntu1804-zseries
-      display_name: "s390x Ubuntu 18.04 (MongoDB 4.4)"
-      batchtime: 1440 # 1 day
-      expansions:
-          build_type: "Release"
-          cmake: "cmake"
-          tar_options: *linux_tar_options
-          cmake_flags: *linux_cmake_flags
-          mongodb_version: *version_44
-      run_on:
-          - ubuntu1804-zseries-small
```

Comment:
Can `ubuntu1804-zseries-small` be replaced with `rhel80-zseries-small`? That was suggested in ticket comments to continue testing with zSeries.

Reply:
I would prefer that to be addressed in a separate PR for [CXX-2492](https://jira.mongodb.org/browse/CXX-2492) instead of this one.